src/styles/_card.scss

```diff
@@ -0,0 +1,135 @@
+@import 'variables';
+
+.card {
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__description {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: $font-size-md;
```

Comment:
Ensure that the variable `$font-size-md` is defined in the 'variables' file. If it's not defined, it will cause a compilation error.